src/scripts/hamburger.js

```diff
@@ -0,0 +1,22 @@
+function burgerNavigation() {
+  const mobileIcon = document.getElementById("mobile-navigation");
+  const mobileBars = mobileIcon.querySelector(".nav-bars");
+  const navigation = document.getElementById("navigation-list");
+  const allLinks = navigation.querySelectorAll("li");
+
+  function toogle() {
+    navigation.classList.toggle("nav-hover");
+    mobileBars.children[0].classList.toggle("rotate-right");
+    mobileBars.children[1].classList.toggle("hidden");
+    mobileBars.children[2].classList.toggle("rotate-left");
```

Comment:
when nav has `nav-hover` class those classes below should be applied. This is correct, but should be implemented with CSS 😄

Reply:
when nav is open those classes below should be applied. This is correct, but should be implemented with CSS